docs/content/get-started/get-playing.md

```diff
@@ -1,6 +1,6 @@
 # Get started: 5 - Get playing!
 
-Back in your SpatialOS [web Console](https://console.improbable.io/projects), you should now see the deployment that you just created appear under your project. Select it to get to the Overview page:
+Back in your SpatialOS [web Console](https://console.improbable.io/projects), you should now see two deployments that you just created appear under your project. Select the one without the `_sim_players` suffix to get to the Overview page:
 
```

Comment:
Just realised we called this "the OVERVIEW screen" in the mobile docs (when we mentioned cloud) (in case Jess spots this). Note to self to standardise "Overview page" is better.

Reply:
cc @jessicafalk ^^^